openapi-spec/swagger.yaml

```diff
@@ -87,7 +92,104 @@ tags:
     externalDocs:
       description: Find out more
       url: 'https://sodafoundation.io/'
+# schemes:
+# - http
 paths:
+  '/v1/{project_id}/aksks':
+    post:
+      tags:
+        - AKSK
+      summary: Create a new AKSK for a User.
```

Comment:
Just thinking, if we are creating a single aksk then why calling apis as aksks (collection)

Reply:
A User can have multiple AKSK .. so its a collection of AKSK for a user_id , hence "aksks"
Also its naming convention of Rest APIs.
To get the AKSKs for any user_id , its /v1/{project_id}/aksks/{user_id} 
which is again, as per convention of Rest. 

https://restfulapi.net/resource-naming/
A resource can be a singleton or a collection. For example, “customers” is a collection resource and “customer” is a singleton resource (in a banking domain). We can identify “customers” collection resource using the URI “/customers”. We can identify a single “customer” resource using the URI “/customers/{customerId}”.